### DISCUSSION TITLE
p8carts directory location

### DISCUSSION THREAD

**Discussion by TheElementalForce** [jtothebell/fake-08#104] [Q&A] [answered]
I just installed fake-08, and i was looking for carts to add to the software. However, after looking in my sd card, there's no "p8carts/" folder as the description says there should be. Am I supposed to just make the folder myself? And if so, where should I put it?

**Comment by jtothebell** [comment, marked as answer]
What platform are you on? It sounds like probably 3DS, Switch, and Wii U, and for those, yes you need to create the p8carts directory in the root of your SD card

**Reply by TheElementalForce** [reply]
Ah, yes im on 3ds, glad i was just misunderstanding and I didn't download it wrong or corrupt it, thanks for the help!
